src/sortStudents.ts

```diff
@@ -1,16 +1,57 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(students: Student[],
+  sortBy: SortType,
+  order: SortOrder): Student[] {
+  let avgGradeA,
+    avgGradeB;
+  const sortedStudents = [...students];
+
+  sortedStudents.sort((a, b) => {
```

Comment:
```suggestion
  sortedStudents.sort((firstStudent, secondStudent) => {
```
be more descriptive :)